### DISCUSSION TITLE
sound not working

### DISCUSSION THREAD

**Discussion by Turok45** [BimmerGestalt/AAIdrive#454] [Q&A]
Hello, i'm going to try some explanation in english, it's not my native language so let's do it. 
Firstly really thank you for this app it's juste great to found the music without touching the phone on road. 
Now i'm going to explain you my problem, at the begining all was perfectly working, but now i have a problem when i made a bleutooth connection whitout AAIdrive, i have music from spotify on my car but without the album cover. 
And when i made music connection with AAIdrive to get spotify on the idrive, it's working i have the album cover i can also found my playlist but i have no sound and the title is playing. i don't know what to do. 
Thank you

**Comment by hufman** [comment]
Are you plugged in to the car with USB or only with Bluetooth?
Is the Advanced Setting to Force Audio Context over USB enabled? This should usually remain disabled, and will almost certainly cause incorrect results on recent versions of Android. If you have an old Android phone that supports USB Audio, it _should_ usually autodetect it and not need the checkbox enabled.
![image](https://user-images.githubusercontent.com/1592375/140455392-4ee22b59-5766-4320-9a61-571b1768a8c0.png)

If your car supports Bluetooth Apps, but if you connect to USB too fast (to charge, maybe), the app connection may go over USB which doesn't usually support audio. If Force Audio Context is enabled, the car will listen to audio over the USB port, but your phone probably will not send audio over USB. Set the car to listen to Bluetooth, and disable Force Audio Focus, and the app should work.

**Reply by Turok45** [reply]
Thank you for your return, i'm going to try this today and will come back to you to inform.

**Reply by Turok45** [reply]
Hello, well it's still not working i don't know what to do, my sound is by bleutooth, but the connection between the phone and the car for AAI is by USB.

**Reply by hufman** [reply]
Did you confirm that the Force Audio Focus switch is turned off? Feel free to clear the app data or uninstall/reinstall, which will force this switch back to the default.
What version of Android do you have? It may be that the app thinks your Android is old enough to support USB audio but the rom doesn't actually include support. On the main tab, can you confirm that the Audio Context checkbox is empty?
![image](https://user-images.githubusercontent.com/1592375/140810590-c1e284be-0afc-42b7-9de6-1ae147cc8fbe.png)
